saskatoon/sitebase/templates/app/base/header.html

```diff
@@ -1,99 +1,125 @@
 {% load static %}
 {% load i18n %}
 {% load auth_users %}
-
 <div class="header-top-area">
     <div class="container">
         <div class="row">
             <div class="col-md-1 col-md-offset-0 col-sm-4 col-sm-offset-4 col-xs-6 col-xs-offset-3">
                 <div class="logo-area">
-                    <a href="/"><img src="{% static 'img/logo.png' %}" alt="Les Fruits Defendus - Saskatoon" /></a>
+                    <a href="/">
+                        <img src="{% static 'img/logo.png' %}"
+                             alt="Les Fruits Defendus - Saskatoon" />
+                    </a>
                 </div>
             </div>
             <div class="col-md-5 col-xs-12">
                 <div class="logo-area">
                     <h2 style="color:white">Les Fruits Défendus</h2>
-                    <p style="color:white"><i>{% translate 'Collectif montréalais de cueillette urbaine de fruits' %}</i></p>
+                    <p style="color:white">
+                        <i>{% translate 'Collectif montréalais de cueillette urbaine de fruits' %}</i>
```

Comment:
If we want this bilingual we should set it in english and mark it for translation.
Alternatively we could remove the translate tags and keep it in french as it's the logo

Reply:
I think logo can be kept in French. It's a nice little touch